### ISSUE TITLE
Let's Start v1.0.7 (or 8) with TW-Sounds!

### ISSUE THREAD

**Issue by flancast90** [flancast90/Speech-To-Text-in-TW5#17]
Hi @BurningTreeC,

Since neither of us have been real active lately in this repo, I've decided to start work on v1.0.7/8, utilising TW-Sounds. I'll add the functions and calls, and if you want, you can look over the code and make it more "tw-like".

**Comment by flancast90** [comment]
Hi @BurningTreeC,
Just wondering if you could push the necessary TW changes so the twsounds library can be accessed from the startup js file. I tried adding a scripts directory to my copy of the plugin, and in it a twsounds.js file (code from https://github.com/flancast90/TW-Sounds/blob/main/src/tw-sounds.js). For some reason, the plugin wouldn't call any of the twsounds functions, even after importing with the (possibly correct tiddlywiki.info file JSON) that @joshuafontany gave

**Comment by BurningTreeC** [comment]
Hi @flancast90 , I haven't had the chance to look at your twsounds library but I guess giving it the field `module-type: library` and requiring it in our startup file could do the trick ...?

**Comment by flancast90** [comment]
That’s exactly what I was doing; After I tried what @joshuafontany, I tried
just adding it the to .info file as a tiddler with module-type: library,
but I became confused with the file structure there, as it seemed to skip
the files folder completely, and go right the the modules/startup.
What @joshuafontany recommended worked except for that it added undefined
to the end of the file name/path.

Any ideas?
